### PR TITLE
Add Channels

### PR DESCRIPTION
**Pull request by MapGuy11**
1. WSNN-LD1
2. WISC-DT2
3. WFMZ-DT2

### CODE REVIEW

**Review by jkemp814 — COMMENTED** [review]
Should we have that dash in the tvg-id field? The EPG might not sit well with that.

**Comment by MapGuy11** [comment]
> Should we have that dash in the tvg-id field? The EPG might not sit well with that.

I honestly don't know how you get your EPG data some sources use the dash like how US OTA does it some don't.

**Review by jkemp814 — COMMENTED** [review]
That link does not work

https://cdn88.theus6tv.tk/united-states/locals/philadelphia/wfmz-dt2.m3u8

**Comment by MapGuy11** [comment]
> That link does not work

Yes I know I am getting weird 500 crashes

**Comment by jkemp814** [comment]
> > Should we have that dash in the tvg-id field? The EPG might not sit well with that.
> 
> I honestly don't know how you get your EPG data some sources use the dash like how US OTA does it some don't.

I use iptv-org EPG repository. Their are no dashes. tvtv.us EPG

**Comment by MapGuy11** [comment]
> > > Should we have that dash in the tvg-id field? The EPG might not sit well with that.
> > 
> > I honestly don't know how you get your EPG data some sources use the dash like how US OTA does it some don't.
> 
> I use iptv-org EPG repository. Their are no dashes. tvtv.us EPG

Ok I will change it tommorow getting late here in the Eastern Time Zone

**Comment by spotanjo3** [comment]
I am still waiting for addition more Portugal channels to https://iptv-org.github.io/iptv/countries/pt.m3u. It is only 41 channels and some didn't worked as I mentioned before. :)

**Comment by MapGuy11** [comment]
> > > Should we have that dash in the tvg-id field? The EPG might not sit well with that.
> > 
> > 
> > I honestly don't know how you get your EPG data some sources use the dash like how US OTA does it some don't.
> 
> I use iptv-org EPG repository. There are no dashes. tvtv.us EPG

@jkemp814 It's all fixed

**Comment by jkemp814** [comment]
I still can't get this link to work:
https://cdn88.theus6tv.tk/united-states/locals/philadelphia/wfmz-dt2.m3u8

Maybe someone else can check it to see of it works on their end?

**Comment by LaneSh4d0w** [comment]
I think I understood what was going on.

@MapGuy11 What's the JS code for WFMZ ?

**Comment by MapGuy11** [comment]
> I think I understood what was going on.
> 
> @MapGuy11 What's the JS code for WFMZ ?

Give me a second I will create an issue on your repo.

File: channels/us.m3u [hunk 1]
```diff
@@ -6615,3 +6615,9 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
 https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-name="Канал Disney" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p)
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="WSNNLD1.us" tvg-name="WSNN-LD1" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/75548/s75548_h3_aa.png" group-title="Locals",WSNN-LD1 (Suncoast News Network)
+https://cdn88.theus6tv.tk/united-states/locals/tampa/wsnn-ld.m3u8
+#EXTINF:-1 tvg-id="WISCDT2.us" tvg-name="WISC-DT2" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/25125/s51307_h3_aa.png" group-title="Locals",WISC-DT2 (My Madison TV)
+https://cdn88.theus6tv.tk/united-states/locals/madison/wisc-dt2.m3u8
+#EXTINF:-1 tvg-id="WFMZDT2.us" tvg-name="WFMZ-DT2" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/35859/s35859_h3_aa.png" group-title="Locals",WFMZ-DT2 (69News Weather Channel) [Offline]
+https://cdn88.theus6tv.tk/united-states/locals/philadelphia/wfmz-dt2.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:6623]:
```suggestion
https://cdn88.theus6tv.tk/united-states/locals/philadelphia/wfmz-dt1.m3u8
```

After verifying @MapGuy11's code.

**Comment by LaneSh4d0w** [comment]
@jkemp814 I've reviewed the code with MapGuy, and we've seen some irregularities in the server we were working on. It should work now. Try on your side.

**Comment by jkemp814** [comment]
Yes it works, good job. 

Sorry it took so long. having to rebuild my workstation crashed.

**Review by jkemp814 — APPROVED** [review]
It works well.

**Pull request merged**